test/performance/commCompare/gbt/many-to-one.ml-compopts

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+# Aim for about 1 second of run time.  The number of operations needed
+# to achieve this varies with the network architecture and op type.
+import os
+
+comm = os.getenv('CHPL_COMM')
+comm_sub = os.getenv('CHPL_COMM_SUBSTRATE')
+ugni = comm == 'ugni'
+gn_aries = comm == 'gasnet' and comm_sub  == 'aries'
+
+if ugni or gn_aries:
+  numGets    =  20000000
+  numPuts    = 100000000
```

Comment:
Interesting -- any idea why puts seem to be much faster than gets?

Reply:
GETs require a PCIe round trip on the remote side to fetch the gotten value, and the network response doesn't happen until after that occurs.  That reduces the rate per initiator, and then having so many requests inbound (numWorkerNodes*numWorkerCores) loads the PCIe as well. 
PUTs don't require that PCIe round trip.  (This is rough; see [here](https://www.cray.com/sites/default/files/resources/CrayXCNetwork.pdf), top of page 22, for more detail.  Also note that we're seeing much more difference than one would expect based on reading that document.  I'm not sure why.)

Reply:
I've just now checked the emitted code for all 5 ops and they're doing what we expect them to do.  So, the much larger observed difference between GETs and PUTs, compared to what's expected, remains unexplained.